lumicks/pylake/tests/conftest.py

```diff
@@ -83,4 +84,9 @@ def h5_file(tmpdir_factory, request):
     mock_file.make_timeseries_channel("Force LF", "Force 1x", [(1, 1.1), (2, 2.1)])
     mock_file.make_timeseries_channel("Force LF", "Force 1y", [(1, 1.2), (2, 2.2)])
 
+    with suppress(NotImplementedError):
```

Comment:
Because that type of channel doesn't exist yet for the v1 file format. I think it's valuable to have the ability to run tests on somewhat realistic test data files that have differen file format versions. (Could even include some actual data files later on?)

I've tried a different way of expressing that intent in the latest push. Not too happy with that version either, but I propose we revisit this part of the code as we get more file format versions. I don't want to end up with dozens of file format version classes, but then again, it seemed like the cleanest way to encode the (sometimes subtle) differences between format versions.

Reply:
Your fixup looks great. Makes it much clearer why that part is being skipped.